.github/helm-values/dev-eu-central-1-alpha.pg-sni-router.yaml

```diff
@@ -6,6 +6,9 @@ exposedService:
   # exposedService.port -- Exposed Service proxy port
   port: 4432
   annotations:
+    service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: ip
+    service.beta.kubernetes.io/aws-load-balancer-scheme: internal
+    service.beta.kubernetes.io/aws-load-balancer-type: external
```

Comment:
What does it mean if type is external and scheme is internal?

Reply:
https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.3/guide/service/annotations/#lb-type

> This annotation specifies the controller used to provision LoadBalancers (as specified in [legacy-cloud-provider](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.3/guide/service/annotations/#legacy-cloud-provider)). Refer to [lb-scheme](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.3/guide/service/annotations/#lb-scheme) to specify whether the LoadBalancer is internet-facing or internal.